### PR TITLE
[86bz26j5z][website] added tips for production

### PR DESCRIPTION
**Pull request by ilyabrower**
I've added lost page about production tips

### CODE REVIEW

File: website/docs/get-started-guide/dev-starter-guide/production-tips.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+Also note that the configuration of the `rule` parameter in JS files uses `enforce: 'pre'`. Otherwise, obfuscation will change the function names and make it impossible to search and replace CSS classes. Add these rules to your `webpack-config.js` as a separate section, since they should process packages from the @semcore library only.
+
+## Server-side rendering
+
+There are two ways to implement SSR.
+
+### SSR via sstyled.getStyles function
+
+This method will work for you if you're not using the @semcore/shadow-loader package (see [CSS extract](/internal/production/#a5c869)).
```
Comment (j-mnizhek) [inline website/docs/get-started-guide/dev-starter-guide/production-tips.md:83]: Where should CSS extract link lead us? Cannot find the appropriate documentation.

File: website/docs/get-started-guide/dev-starter-guide/production-tips.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+Use the @semcore/ui/core package and the `sstyled.getStyles` function which will return `style` tags with all the necessary styles:
```
Comment (j-mnizhek) [inline website/docs/get-started-guide/dev-starter-guide/production-tips.md:85]: BTW, shouldn't we change @semcore/ui/core there to intergalactic?

**Review by j-mnizhek — APPROVED** [review]
Left two small comments, fixed documentation a little bit. 

**We need to fix links to CSS extract in the doc.** 

Everything other looks legit 👍

File: website/docs/get-started-guide/dev-starter-guide/production-tips.md (hunk 1 above)
Comment (ilyabrower) [reply website/docs/get-started-guide/dev-starter-guide/production-tips.md]: fixed

File: website/docs/get-started-guide/dev-starter-guide/production-tips.md (hunk 2 above)
Comment (ilyabrower) [reply website/docs/get-started-guide/dev-starter-guide/production-tips.md]: fixed

**Pull request merged**